packages/space-opera/src/components/mobile_view/mobile_view.ts

```diff
@@ -146,15 +149,19 @@ export class MobileView extends LitElement {
       this.initializeToast(json.updatedContent);
       setTimeout(() => {
         this.toastClassName = '';
-      }, 5000);
+      }, 7000);
       await this.waitForData(json);
     } else {
       console.error('Error:', response);
     }
   }
 
   async triggerFetchLoop() {
-    await this.fetchLoop();
+    try {
```

Comment:
fixes issue where the fetch loop timing out would crash the loop causing the mobile device to close its "port"